clone/cmd/sumdbclone/README.md

```diff
@@ -13,6 +13,34 @@ go run ./clone/cmd/sumdbclone --alsologtostderr --v=1 --mysql_uri 'clonetool:let
 
 ## Docker
 
+### SQL DB in Docker
```

Comment:
Is it worth adding a pointer to the `rpi` docker-compose file? (or is there some magic in `docker-compose` which somehow does the right thing?)

Reply:
Updated this to use `docker compose` instead of `docker-compose` which uses a newer version (written in Go instead of Python), which supports the healthcheck syntax on rpi. It still doesn't do anything magical with the override file, so this is now documented explicitly.